### ISSUE TITLE
[Stepper / Stepper Item]: Ensure implicit list markup is avoided (CAL-AMP34) 

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-components#13537]
**monday.com sync:** #10821963626

### Violation: 
 Ensure implicit list markup is avoided 

### Recommended solution
Setup markup on the components that fit either an unordered list or ordered list (if numbered). For instance:

```html
<!-- When the component is *not* `numbered` -->
<ul><!--Stepper-->
  <li></li><!--Stepper item-->
</ul>

<!-- When the component is `numbered` -->
<ol><!--Stepper-->
  <li></li><!--Stepper item-->
</ol>
```

### WCAG Reference: 
- [1.3.1 Info and Relationships](https://www.w3.org/WAI/WCAG22/Understanding/info-and-relationships.html) 

### Severity: 
 Low 

### Media Type: 
 Lists 

--- 
### Areas for Remediation: 
- Title: CAL-AMP34 Ensure implicit list markup is avoided 
- Module: M5 Stepper, Tabs, Link, FAB 
- Finding ID: [CAL-AMP34](https://esri.hub.essentia11y.com/short-link/tdJwlO19iIQ4YoCR) 

**Issue**
There is content that appears as a list but does not use list markup. Examples include:
- Add info
- Add data
- Add images
- Review.

**User Impact**
Screen reader users will have difficulty understanding that this content is a list.

**Code Reference** 
 ```
<calcite-stepper icon="" class="demo-component" numbered="" role="region" layout="horizontal" scale="m" aria-label="Progress steps" calcite-hydrated="">
(...)</calcite-stepper>

```

**Steps to Reproduce**
1. Locate the list.
2. Inspect it with Chrome DevTools.
3. Note that it does not use ```<ul>```, ```<ol>```, ```<dl>``` and related elements.
 
**Recommendation**
Ensure implicit list markup is avoided. Lists must use ```<ul>```, ```<ol>```, ```<li>```, ```<dl>```, ```<dt>```, and ```<dd>``` elements as appropriate.

In this case, the developer should provide ```<ol>``` list markup to the mentioned instances.

**Compliant Code Example**
```
<!-- unordered list example -->
<ul>
<li>apples</li>
 <li>oranges</li>
 <li>strawberries</li>
 <li>pears</li> 
</ul>
---
<!-- ordered list example -->
<ol>
 <li>First step in the process</li>
 <li>Second step in the process</li>
 <li>Third step in the process</li>
 <li>Fourth step 4 in the process</li>
 </ol>
```

 **Issue Image** 
 https://desktopqa/aic_lap/Calcite/Calcite2025/CAL-AMP34.png 
 

 --- 
### Additional Resources: 
- [LA Platform Report Link](https://esri.hub.essentia11y.com/short-link/tdJwlO19iIQ4YoCR) 
- [Esri Accessibility Resources](https://esriis.sharepoint.com/sites/Accessibility)
- Report Source: Product accessibility evaluation conducted on Calcite by Level Access and delivered August, 2025.
